ndcube/ndcube.py

```diff
@@ -186,11 +187,22 @@ def __init__(self, data, wcs=None, uncertainty=None, mask=None, meta=None,
             raise TypeError("The extra_coords argument must be a ndcube.ExtraCoords object.")
 
         self._extra_coords = extra_coords
+        self._global_coords = GlobalCoords(self)
 
     @property
     def extra_coords(self):
+        """
+        An `.ExtraCoords` object holding extra coordinates aligned to array axes.
+        """
         return self._extra_coords
 
+    @property
```

Comment:
Do we want to expose this directly so users can add and remove coords themselves?

Reply:
they can, this will return the actual object by reference. What this is doing is making a single object always be associated with the NDCube. i.e. you can't set the `.global_coords` property, you can only mutate it in place.

Reply:
:+1: